src/helpers/numberFormat.ts

```diff
@@ -0,0 +1,6 @@
+export function numberFormatter(language: string | undefined, humanize: boolean, round: boolean) {
+  return new Intl.NumberFormat(language, {
+    ...(humanize ? { notation: 'compact', compactDisplay: 'short' } : {}),
```

Comment:
Thanks for tidying this up, was a bit lazy repeating the formatters in the various files.

Reply:
Yeah, I was going the lazy route too but Intl.Number... lines are so much longer than d3.format and I had to update the &*(^ typescript definitions so that pushed me to consolidation :)